.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\packages",
+    "\\packages\\TorneloScoresheet",
+    "\\packages\\TorneloScoresheet\\src"
+  ],
+  "SelectedNode": "\\packages\\TorneloScoresheet\\src\\arbiterSetupMode.ts",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
Don't know what any of this is - but I don't think the `.vs` directory is meant to be in this PR? I think this is some editor config or smt that only applies to your particular editor setup?

Reply:
So I think `.vs` should be in our `.gitignore`?